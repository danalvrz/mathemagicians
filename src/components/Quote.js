import React from 'react';
import Header from './Header';

const Quote = () => (
  <div>
    <Header />
    <section className="quotePage">
      <blockquote className="quote" style={{ color: 'aliceblue' }}>
        <span className="quoteText">&quot;Avoid negativity, use Math.abs().&quot;</span>
        {' '}
        <br />
        - Just another alge&apos;bro
      </blockquote>
    </section>
  </div>
);

export default Quote;
