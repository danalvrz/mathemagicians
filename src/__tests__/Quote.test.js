import React from 'react';
import renderer from 'react-test-renderer';
// eslint-disable-next-line no-unused-vars
import { render, fireEvent } from '@testing-library/react';
import { HashRouter as Router } from 'react-router-dom';
import Quote from '../components/Quote';

test('renders correctly', () => {
  const tree = renderer.create(
    <Router>
      <Quote />
    </Router>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
