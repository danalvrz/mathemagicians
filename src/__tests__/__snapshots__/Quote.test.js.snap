// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className="menuContainer"
  >
    <h1>
      Mathemagicians
    </h1>
    <nav
      className="menuList"
    >
      <a
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      |
      <a
        href="#/calculator"
        onClick={[Function]}
      >
        Cowculator
      </a>
      |
      <a
        href="#/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </nav>
  </header>
  <section
    className="quotePage"
  >
    <blockquote
      className="quote"
      style={
        Object {
          "color": "aliceblue",
        }
      }
    >
      <span
        className="quoteText"
      >
        "Avoid negativity, use Math.abs()."
      </span>
       
      <br />
      - Just another alge'bro
    </blockquote>
  </section>
</div>
`;
